src/main/java/com/minecolonies/blockout/views/DropDownList.java

```diff
@@ -0,0 +1,340 @@
+package com.minecolonies.blockout.views;
+
+import com.minecolonies.coremod.util.Log;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.util.Log'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)